docs/pbc/all/product-information-management/latest/marketplace/install-and-upgrade/install-features/install-the-merchant-portal-data-import-feature.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Install the Marketplace Merchant Portal Data Import feature
+last_updated: Jul, 2025
+description: This document describes the process how to integrate the the Marketplace Merchant Portal Data Import feature into a Spryker Marketplace based project.
+template: feature-integration-guide-template
+---
+
+{% include pbc/all/install-features/latest/marketplace/install-the-merchant-portal-data-import-feature.md %}
+
+
+## Next steps
+
+[Install merchant products data import](/docs/pbc/all/product-information-management/latest/marketplace/install-and-upgrade/install-features/install-the-merchant-product-data-import-feature).
```

Comment:
Should this look like the standard related features section? 
https://docs.spryker.com/docs/pbc/all/merchant-management/latest/marketplace/install-and-upgrade/install-features/install-the-marketplace-merchant-portal-core-feature#update-navigation

Reply:
There are two separate features IMO
1. Generic data import page for merchant portal
2. extension for this feature, which adds product data import functionality to the 1)

Later, we expect more data imports to be supported on this page.

But I'm OK if we include all of them on this page. What do you think @andriitserkovnyi ?